packages/frontend/src/js/app/store/index.js

```diff
@@ -167,8 +167,14 @@ export default new Vuex.Store({
         },
 
         async addTownActor({ commit }, { townId, actor }) {
-            const { actors } = await addActor(townId, actor);
-            commit("updateShantytownActors", { townId, actors });
+            const response = await addActor(townId, actor);
+
```

Comment:
Si j'ai bien compris, response.actors est undefined quand on invite un autre utilisateur que celui connecté ? Je pense que ça vaut le coup de rajouter un commentaire


```suggestion
// response.actors is not defined when an actor other than the connected user is added (we send an email to get his confirmation instead)
```